grammar.js

```diff
@@ -1175,8 +1175,7 @@ module.exports = grammar({
 
     unsigned_literal: $ => seq(
       choice($.integer_literal, $.hex_literal, $.bin_literal),
-      /[uU]/,
-      optional("L")
+      /[uU][L]?/
```

Comment:
I think we could shorten this to

```suggestion
      /[uU]L?/
```